### PR TITLE
Fix vulnerable gem versions

### PR DESCRIPTION
**Pull request by dgmstuart**
The following gems had vulnerabilities:

- loofah (CVE-2018-8048, before 2.2.1)
- nokogiri (CVE-2017-18258, before 1.8.2)
- ruby-rack-protection (CVE-2018-1000119, before 1.5.5)
- rails-html-sanitizer (CVE-2018-3741, before 1.0.4)

Previously the approach was to update everything, so I've done the same.

The gem server runs fine locally (`rackup`), so I assume everything is ok (?)

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -13,91 +13,92 @@ GIT
 GEM
   remote: https://rubygems.org/
   specs:
-    actionpack (4.2.10)
```
Comment (teoljungberg) [inline Gemfile.lock]: Jumping from 4.2 to 5. 2seems like it could cause issues. Is there another patch level we can use that has the CVE's fixed?

**Comment by teoljungberg** [comment]
@dgmstuart You updated _all_ gems in this PR, not just the ones that had CVEs. Can we only update those to minimze the noise?

**Comment by dgmstuart** [comment]
@teoljungberg sure - can do. I was following the pattern of what had been done before in 057bc9f, but a minimal PR makes sense too.

**Comment by dgmstuart** [comment]
@teoljungberg how about now?

**Pull request merged**